docs/scripts/api/buildInterfacesDocumentation.ts

```diff
@@ -112,13 +112,8 @@ const parseInterfaceSymbol = (
 
       const exportedSymbol = project.exports[interfaceName];
       const type = project.checker.getDeclaredTypeOfSymbol(exportedSymbol);
-      const typeDeclaration = type.symbol.declarations?.[0];
```

Comment:
I had to remove `typeDeclaration` check since `GridColDef` is now `TypeAlias`, not `Interface`.
Otherwise it was throwing an error here.